nttt/project.py

```diff
@@ -0,0 +1,89 @@
+import os
```

Comment:
`project.py` might be a confusing name since I don't see anything here related to projects. It contains file system utility functions so I would consider moving them to `utilities.py`

Reply:
The first name of the module was file_system. This sounded far too generic. The next name was contents... no good either. And then it became project, because it is about the files and folders of what ultimately becomes a project on the Raspberry Pi website. Maybe moving it to the utilities module is the best way to end this struggle ;-)